imageviewer.js

```diff
@@ -4,22 +4,24 @@
     Copyright (c) 2015-2016 to Sudhanshu Yadav - ignitersworld.com , released under the MIT license.
     Demo on: http://ignitersworld.com/lab/imageViewer.html
 */
-
 /*** picture view plugin ****/
-(function ($, window, document, undefined) {
-    "use strict";
+(function(window, document, undefined) {
+    'use strict'
```

Comment:
Let's keep the line ending colon across files.
Also, can we use 2 space indentation?